Doc/tools/templates/layout.html

```diff
@@ -1,4 +1,14 @@
 {% extends "!layout.html" %}
+{% block header %}
+{%- if outdated %}
+<div id="outdated-warning" style="padding: .5em; text-align: center; background-color: #FFBABA; color: #6A0E0E;">
+    {% trans %}This document is for an old version of Python that is {% endtrans %}<a href="https://devguide.python.org/devcycle/#end-of-life-branches">{% trans %}no longer supported{% endtrans %}</a>.
+    {% trans %}You should upgrade, and read the {% endtrans %}
+    <a href="/{{ language + '/' if language and language != 'en' else '' }}3/{{ pagename }}{{ file_suffix }}">{% trans %} Python documentation for the current stable release{% endtrans %}</a>.
```

Comment:
This href needs to be an absolute URL to the website and not just relative because the docs are also produced as a downloadable set of files for a particular release and thus need to work when browsing via files.  Try building the docs and then opening the built file directly in a browser (for example. on macOS, if you were in the Doc directory, you could just type 'open build/html/index.html')